src/main/java/jp/ikedam/jenkins/plugins/jobcopy_builder/AbstractXmlJobcopyOperation.java

```diff
@@ -173,14 +176,21 @@ public void fatalError(SAXParseException exception)
                 exception.printStackTrace(logger);
             }
         });
-        InputStream is = new ByteArrayInputStream((encoding != null)?
-                xmlString.getBytes(encoding)
-                :xmlString.getBytes()
-                ); 
+        InputStream is = createInputStreamFromString(xmlString, encoding);
         
         return builder.parse(is);
     }
     
+    @SuppressFBWarnings("DM_DEFAULT_ENCODING")
+    private InputStream createInputStreamFromString(String str, @CheckForNull String encoding)
+            throws UnsupportedEncodingException
+    {
+        return new ByteArrayInputStream((encoding != null)?
+            str.getBytes(encoding)
+            :str.getBytes()
+        ); 
+    }
+    

```

Comment:
<BugInstance type='DM_DEFAULT_ENCODING' priority='High' category='I18N' message='Found reliance on default encoding in jp.ikedam.jenkins.plugins.jobcopy_builder.AbstractXmlJobcopyOperation.getXmlDocumentFromString(String, String, PrintStream): String.getBytes()' lineNumber='176'/>